### ISSUE TITLE
subvolume supervision solve overfitting

### ISSUE THREAD

**Issue by niexiaotian** [yangyanli/PointCNN#23]
sorry,i want to know why the subvolume supervision can solve overfitting,i think,it only realizes the prediction of a whole through a part.

**Comment by yangyanli** [comment]
When only partial observations are presented to the network, it is pushed to learn harder in training, thus can generalize better on testing. It is conceptually similar to Dropout.

**Comment by niexiaotian** [comment]
thank you!

**Comment by niexiaotian** [comment]
thank you for your answer
